docs/database/cosmosdb.md

```diff
@@ -63,12 +63,11 @@ Semantic conventions described in this document apply to the call-level spans on
 | [`azure.client.id`](/docs/attributes-registry/azure.md) | string | The unique identifier of the client instance. | `3ba4827d-4422-483f-b59f-85b74211c11d`; `storage-client-1` | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
 | [`azure.cosmosdb.request.body.size`](/docs/attributes-registry/azure.md) | int | Request payload size in bytes. |  | `Recommended` | ![Development](https://img.shields.io/badge/-development-blue) |
 | [`db.operation.batch.size`](/docs/attributes-registry/db.md) | int | The number of queries included in a batch operation. [9] | `2`; `3`; `4` | `Recommended` | ![Release Candidate](https://img.shields.io/badge/-rc-mediumorchid) |
-| [`db.query.summary`](/docs/attributes-registry/db.md) | string | Low cardinality representation of a database query text. [10] | `SELECT wuser_table`; `INSERT shipping_details SELECT orders`; `get user by id` | `Recommended` [11] | ![Release Candidate](https://img.shields.io/badge/-rc-mediumorchid) |
```

Comment:
query syntax in cosmos is only used to refine items when querying from specific container, i.e. it only contains stuff we would not put into the summary. Operation and collection name come from the structured part of it and not from the query text.

So removing summary from cosmos, and also making query text not sampling relevant.

Reply:
> query syntax in cosmos is only used to refine items when querying from specific container

maybe this should be stored somewhere else besides `db.query.text`?

my concern is that when people see `db.query.text` they will think that represents the whole query as opposed to part of the query

(this discussion can definitely be split out into an issue since cosmos isn't in the initial stability wave)

Reply:
filed https://github.com/open-telemetry/semantic-conventions/issues/2213 and added a comment on cosmos yaml so we don't forget to follow up when cosmos goes stable